### ISSUE TITLE
[BUG]: HistoricalCryptoTradesRequest from / into behaviour seems flipped?

### ISSUE THREAD

**Issue by matsydoodles** [alpacahq/alpaca-trade-api-csharp#434]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Just giving the Crypto API a bash and using the same sort of code I was using successfully for the pre-existing. 

However when I use `HistoricalCryptoTradesRequest` as I did with `HistoricalTradesRequest` I get an exception thrown `"Time interval should be valid. (Parameter 'from')"`.

If it flip my `From` value to `Into` and my `Into` to `From` it works and the `Interval` is reporting the expected values!

```
at Alpaca.Markets.TimeInterval.Inclusive..ctor(Nullable`1 from, Nullable`1 into)
   at Alpaca.Markets.TimeInterval.GetInclusive(Nullable`1 from, Nullable`1 into)
   at Alpaca.Markets.HistoricalRequestBase..ctor(IEnumerable`1 symbols, DateTime from, DateTime into)
   at Alpaca.Markets.HistoricalCryptoRequestBase..ctor(IEnumerable`1 symbols, DateTime from, DateTime into)
   at Alpaca.Markets.HistoricalCryptoTradesRequest..ctor(IEnumerable`1 symbols, DateTime from, DateTime into)
   at Alpaca.Markets.HistoricalCryptoTradesRequest..ctor(String symbol, DateTime from, DateTime into)
```

### Expected Behavior

`From` is the lower `DateTime` value and `Into` is the greater `DateTime` value of a `HistoricalCryptoTradesRequest` as per `HistoricalTradesRequest`.

### Steps To Reproduce

```markdown
var from = new DateTime(2020, 11, 1);
var into = new DateTime(2020, 11, 7);

_ = new HistoricalCryptoTradesRequest(dataRequest.Symbol, from, into);
```

### Environment

```markdown
- SDK Version: 5.2.3
- OS (version, bitness): 10.0.19042 Build 19042
- .NET SDK (version): 
- target process .NET version/bitness: .NET Core 3.1
```

**Comment by OlegRa** [comment]
@matsydoodles Please, test it in SDK version [5.2.5](../releases/tag/sdk-5.2.5) or later and close if the problem is fixed.

**Comment by matsydoodles** [comment]
@OlegRa was fixed in [5.2.6](https://github.com/alpacahq/alpaca-trade-api-csharp/releases/tag/sdk-5.2.6) and looks good cheers!
